ext/mixed/display-templates.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html><html><head></head><body>
+
+<template id="term-entry-template"><div class="entry" data-type="term">
+    <div class="entry-header1">
```

Comment:
Add the class `expression-scan-toggle` or change the `ignoreNodes` class name to `entry-header1` in `search-frontend.js` and `popup-nested.js`

Reply:
`term-expression-text` more directly corresponds to the old `expression-scan-toggle`; do you think the entire header should be omitted also? I think in practice, the effect will be the same unless there is Japanese text in the tags or something.

Reply:
I think it's fine the way you did it, because this feature is mainly useful for preventing a popup from showing up when you try to click a kanji. It also makes sense that `.entry-header1` encapsulates both the expressions and frequencies.

Reply:
Actually it seems that another one is needed for `.kanji-glyph` below

Reply:
Updated to `source-text` style and added it to both the relevant templates. This should make it easier to extend in the future if necessary.